web/html/forms-ii/radio-buttons.md

```diff
@@ -17,7 +17,7 @@ aspects:
   - workout
 
 links:
-  - '[MDN docs for radio buttons](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio){website}'
+  - '[MDN docs for radio buttons](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio){documentation}'
```

Comment:
can you do a grep & replace for all the instances of "MDN docs" and remove them? Also tweak the titles a bit. One way you could do it is to make the description be close to the title of the article/documentation/discussion

```suggestion
  - '[MDN docs for radio buttons](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio){documentation}'
```